### DISCUSSION TITLE
Db error: Invalid default value for 'date_ordered' Error code:1067

### DISCUSSION THREAD

**Discussion by kvirtu** [givanz/Vvveb#13] [General]
Db error: Invalid default value for 'date_ordered' Error code:1067

**Comment by givanz** [comment]
Thanks for the bug report.
I fixed it, please redownload the zip and see if it works now.
It looks that on older Mysql/MariaDb versions NOW() is not supported as default value.

**Comment by kvirtu** [comment]
hoster use mysql: 5.7.40-log - Source distribution
now this error

✖ Db error: Incorrect string value: '\x98oarec...' for column 'name' at row 44 INSERT INTO `product_category_description` VALUES (1,1,'Monitors','Monitors','Monitors','Monitors','',''), (2,1,'Cameras','Cameras','Cameras','Cameras','',''), (3,1,'Web Cameras','Web Cameras','Web Cameras','Web Cameras','',''), (4,1,'Scanners','Scanners' in product_category/product_category_description.insert Error code:1366

**Open the address of your website (eg: [localhost](http://localhost/)) and the installation wizard will show** - the installation does not start automatically. only through http://localhost/install

**Comment by givanz** [comment]
This was caused by some non ascii characters in the demo content insert, I changed them to ascii and updated the zip.
I will try to install mysql 5.7 version and see if there are other issues, I only tested with the more common 8.0 version and MariaDB 10.3.37.

There should be an automatic redirect to localhost/install on first run I suspect there is a php error that prevents the redirect and if error reporting or show error is off then no message is displayed.
Can you please check php error log and see if there is an error message? A common cause is missing php xml/libxml/dom extensions, I just added some special checks for these extensions to avoid this issue.

Thanks for your feedback.

**Comment by kvirtu** [comment]
Thank you, no errors during installation. But, after installation, a white screen with a code. 
http://mysite/admin/?success=Installation%20succesful!
`<html><head></head><body></body></html>`
from phpinfo:
PHP Version 8.0.26
'./configure' '--disable-all' '--program-prefix=' '--with-config-file-scan-dir=/usr/local/etc/php' '--with-layout=GNU' '--with-libxml' '--with-openssl' '--with-password-argon2=/usr/local' '--enable-dtrace' '--enable-embed' '--enable-fpm' '--with-fpm-user=www' '--with-fpm-group=www' '--enable-mysqlnd' '--prefix=/usr/local' '--localstatedir=/var' '--mandir=/usr/local/man' '--infodir=/usr/local/share/info/' '--build=amd64-portbld-freebsd13.1' 'build_alias=amd64-portbld-freebsd13.1' 'PKG_CONFIG=pkgconf' 'PKG_CONFIG_LIBDIR=/wrkdirs/usr/ports/lang/php80/work/.pkgconfig:/usr/local/libdata/pkgconfig:/usr/local/share/pkgconfig:/usr/libdata/pkgconfig' 'CFLAGS=-O2 -pipe -fstack-protector-strong -fno-strict-aliasing ' 'LDFLAGS=-L/usr/lib -lcrypto -lssl -fstack-protector-strong ' 'CPP=cpp' 'CXXFLAGS=-O2 -pipe -fstack-protector-strong -fno-strict-aliasing ' 'OPENSSL_CFLAGS=-I/usr/include' 'OPENSSL_LIBS=-L/usr/lib -lssl -lcrypto'

**Reply by givanz** [reply]
This looks like a php error that is not displayed because error display is turned off.
Can you please check php error log (usually located at /var/log/php8_errors.log) and copy paste the last lines?

**Reply by kvirtu** [reply]
php8_errors.log - no errors
From error.log
[Wed Dec 21 15:40:27.625097 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: SELECT value: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.625222 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215:             FROM `option` AS _: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.625315 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: \t\tWHERE _.`key` = "insert-scripts.header": /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.626212 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: SELECT value: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.626293 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215:             FROM `option` AS _: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.626384 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: \t\tWHERE _.`key` = "insert-scripts.footer": /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633092 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: SELECT * : /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633150 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: \t\t\tFROM post AS _: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633324 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: \t\t\tLEFT JOIN post_description pd ON (_.post_id = pd.post_id AND pd.language_id = 1)  : /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633352 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: \t\tWHERE : /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633371 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: \t\t\t: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633392 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: \t\t\t1: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633405 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: : /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633446 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215:             \t\t : /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633520 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: \t\t\t\tAND pd.slug = "favicon.ico" : /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633550 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215:         \t: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633628 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215:                 AND _.type = "post": /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633664 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215:         \t\t\t\t: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633677 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: : /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.633718 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215:         LIMIT 1: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.634547 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215:  SELECT `key` as array_key,`value` as array_value FROM post_meta as _: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.634618 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: \t\t\tWHERE _.post_id = NULL: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.634779 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: SELECT `key` as array_key,`value` as array_value FROM post_meta as _: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.634841 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: \t\t\tWHERE _.post_id = NULL: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.677214 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: PHP Warning:  Undefined property: Vvveb\\System\\Core\\View::$user in /virt/homes/vpu2/htdocs/storage/compiled-templates/app_1_landing_error404.html on line 223: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.677552 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: PHP Warning:  Trying to access array offset on value of type null in /virt/homes/vpu2/htdocs/storage/compiled-templates/app_1_landing_error404.html on line 223: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.677861 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: PHP Warning:  Undefined property: Vvveb\\System\\Core\\View::$cart in /virt/homes/vpu2/htdocs/storage/compiled-templates/app_1_landing_error404.html on line 353: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/
[Wed Dec 21 15:40:27.678189 2022] [cgi:error] [pid 54790] [client 31.144.37.162:37271] AH01215: PHP Warning:  Trying to access array offset on value of type null in /virt/homes/vpu2/htdocs/storage/compiled-templates/app_1_landing_error404.html on line 353: /virt/homes/vpu2/cgi-bin/php, referer: http://mysite/admin/

**Comment by givanz** [comment]
Thanks for the log, it looks like there is a 404 not found page displayed this usually happens if mod_rewrite is not configured for apache or if you are using nginx then the configuration is missing.

If you are using apache make sure that it uses .htaccess (the file is located in / and /public) and that mod_rewrite is installed.

If you are using nginx then use nginx.txt (included in the zip) and just change the server_name and paths to point to correct root location and change `fastcgi_pass` to use your version of php.

It is recommended to use /public folder for root/htdocs for better security instead of / to avoid exposing php files, both folders include .htaccess

**Comment by kvirtu** [comment]
WebServer: Server version: Apache/2.4.54 (FreeBSD)
Perl: This is perl 5, version 32, subversion 1 (v5.32.1) built for amd64-freebsd-thread-multi
PHP: PHP 8.0.26 (cgi-fcgi) (built: Dec 1 2022 05:01:22)
MySQL: mysqld Ver 5.7.40 for FreeBSD13.1 on amd64 (Source distribution)

mod_rewrite - supported

File **php8_errors.log** a created manually 
31.144.37.162 - - [21/Dec/2022:20:33:11 +0200] "GET /admin/ HTTP/1.1" 200 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36"
31.144.37.162 - - [21/Dec/2022:20:33:11 +0200] "GET /favicon.ico HTTP/1.1" 404 17933 "http://mysite/admin/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36"

**Comment by kvirtu** [comment]
**php8_errors.log** 
[21-Dec-2022 21:09:33 Europe/Kiev] PHP Warning:  ini_set(): Session ini settings cannot be changed when a session is active in /virt/homes/vpu2/htdocs/system/session/php.php on line 36
[21-Dec-2022 21:09:33 Europe/Kiev] PHP Warning:  ini_set(): Session ini settings cannot be changed when a session is active in /virt/homes/vpu2/htdocs/system/session/php.php on line 37
[21-Dec-2022 21:09:33 Europe/Kiev] PHP Warning:  ini_set(): Session ini settings cannot be changed when a session is active in /virt/homes/vpu2/htdocs/system/session/php.php on line 38
[21-Dec-2022 21:09:33 Europe/Kiev] PHP Warning:  ini_set(): Session ini settings cannot be changed after headers have already been sent in /virt/homes/vpu2/htdocs/system/session/php.php on line 39
[21-Dec-2022 21:09:33 Europe/Kiev] PHP Notice:  session_start(): Ignoring session_start() because a session is already active in /virt/homes/vpu2/htdocs/system/session/php.php on line 41
[21-Dec-2022 21:09:33 Europe/Kiev] SELECT value
             FROM `option` AS _
   WHERE _.`key` = "insert-scripts.header"
[21-Dec-2022 21:09:33 Europe/Kiev] SELECT value
             FROM `option` AS _
   WHERE _.`key` = "insert-scripts.footer"
[21-Dec-2022 21:09:33 Europe/Kiev] SELECT *
    FROM post AS _
    LEFT JOIN post_description pd ON (_.post_id = pd.post_id AND pd.language_id = 1)  
   WHERE
    
    1

                
     AND pd.slug = "favicon.ico"
          
                 AND _.type = "post"
             

         LIMIT 1
[21-Dec-2022 21:09:33 Europe/Kiev]  SELECT `key` as array_key,`value` as array_value FROM post_meta as _
    WHERE _.post_id = NULL
[21-Dec-2022 21:09:33 Europe/Kiev] SELECT `key` as array_key,`value` as array_value FROM post_meta as _
    WHERE _.post_id = NULL
[21-Dec-2022 21:09:33 Europe/Kiev] PHP Warning:  Cannot modify header information - headers already sent by (output started at /virt/homes/vpu2/htdocs/system/session/php.php:36) in /virt/homes/vpu2/htdocs/system/core/frontcontroller.php on line 95
[21-Dec-2022 21:09:33 Europe/Kiev] PHP Warning:  Undefined property: Vvveb\System\Core\View::$user in /virt/homes/vpu2/htdocs/storage/compiled-templates/app_1_landing_error404.html on line 223
[21-Dec-2022 21:09:33 Europe/Kiev] PHP Warning:  Trying to access array offset on value of type null in /virt/homes/vpu2/htdocs/storage/compiled-templates/app_1_landing_error404.html on line 223
[21-Dec-2022 21:09:33 Europe/Kiev] PHP Warning:  Undefined property: Vvveb\System\Core\View::$cart in /virt/homes/vpu2/htdocs/storage/compiled-templates/app_1_landing_error404.html on line 353
[21-Dec-2022 21:09:33 Europe/Kiev] PHP Warning:  Trying to access array offset on value of type null in /virt/homes/vpu2/htdocs/storage/compiled-templates/app_1_landing_error404.html on line 353
31.144.37.162 - - [21/Dec/2022:21:09:33 +0200] "GET /favicon.ico HTTP/1.1" 200 19090 "http://mysite/admin/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36"

**Comment by givanz** [comment]
I found a bug that checks frontend routes on admin also that might cause this issue, please redownload zip and update files, no need to reinstall.

If it still doesn't work please also check that the configuration of your virtual host allows loading .htaccess, it should contain `AllowOverride All` for the Vvveb root directory.

```
<Directory />
    AllowOverride All
</Directory>
```

**Comment by kvirtu** [comment]
31.144.37.162 - - [21/Dec/2022:21:58:35 +0200] "GET /admin/ HTTP/1.1" 304 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36"
[21-Dec-2022 21:58:36 Europe/Kiev] PHP Notice:  session_start(): Ignoring session_start() because a session is already active in /virt/homes/vpu2/htdocs/system/session/php.php on line 41
[21-Dec-2022 21:58:36 Europe/Kiev] SELECT *
    FROM post AS _
    LEFT JOIN post_description pd ON (_.post_id = pd.post_id AND pd.language_id = 1)  
   WHERE
    
    1

                
     AND pd.slug = "favicon.ico"
          
                 AND _.type = "post"
             

         LIMIT 1
[21-Dec-2022 21:58:36 Europe/Kiev]  SELECT `key` as array_key,`value` as array_value FROM post_meta as _
    WHERE _.post_id = NULL
[21-Dec-2022 21:58:36 Europe/Kiev] SELECT `key` as array_key,`value` as array_value FROM post_meta as _
    WHERE _.post_id = NULL
[21-Dec-2022 21:58:36 Europe/Kiev] PHP Warning:  Cannot modify header information - headers already sent by (output started at /virt/homes/vpu2/htdocs/system/session/php.php:41) in /virt/homes/vpu2/htdocs/system/core/frontcontroller.php on line 95
31.144.37.162 - - [21/Dec/2022:21:58:35 +0200] "GET /favicon.ico HTTP/1.1" 200 18030 "http://mysite/admin/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36"

**if add** in  .htaccess
```
<Directory />
    AllowOverride All
</Directory>
```
**error 500 - Directory not allowed here**

**Comment by givanz** [comment]
`AllowOverride All` must be added to `<Directory>` in a apache virtual host configuration file.
For example the default apache virtual host is usually located at /etc/apache2/sites-available/000-default.conf

```
<VirtualHost *:80>
	#ServerName www.example.com
	DocumentRoot /var/www/html
</VirtualHost>
```

Should be changed to 

```
<VirtualHost *:80>
	#ServerName www.example.com
	DocumentRoot /var/www/html
	
	<Directory /var/www/html>
		Options FollowSymLinks
		Allow from all
		AllowOverride All
	</Directory>
</VirtualHost>
```

I made a quick test with apache2 and php8.2 on Ubuntu 20.3 using for /etc/apache2/sites-available/000-default.conf

```
<VirtualHost *:80>
	#ServerName www.vvveb.com
	DocumentRoot /var/www/html/vvveb/public

    <Directory />
      Options FollowSymLinks
      AllowOverride All
    </Directory>

	<Directory /var/www/html/vvveb/public>
        Options Indexes FollowSymLinks
        AllowOverride All
    </Directory>
	
</VirtualHost>
```

and .htaccess is loaded and mod rewrite working.

Make sure that mod rewrite is enabled by running
`sudo a2enmod rewrite`

To get rid of the session already started warning in the logs update /system/session/php.php from zip, this warning is strange and I can't reproduce it.

**Comment by kvirtu** [comment]
I can't edit apache conf files .
how to add **AllowOverride All** in  .htaccess so that there is no error 500: **`/virt/homes/vpu2/htdocs/.htaccess: <Directory not allowed here`**

**Reply by givanz** [reply]
`AllowOverride` can't be added to .htaccess it needs to be added to virtual host config.

If you get error 500 (internal sever error) when changing .htaccess it means that .htaccess is loaded and that AllowOverride is already enabled.

**Comment by kvirtu** [comment]
I recreated the database , reloaded zip-archive, and try reinstall,but errors 
[https://ibb.co/KNmyNKc](url) ( PHP Notice:  session_start(): Ignoring session_start() because a session is already active in /virt/homes/vpu2/htdocs/system/session/php.php on line 41 )

P.S. I think you should try install on a shared hosting, where  is no way to edit the configs of apache.

**Reply by givanz** [reply]
If you are using shared hosting then probably mod rewrite and AllowOverride are already enabled.
In the last update I changed the code to start session only if it's not active to avoid this notice but if it's still not working please edit system/session/php.php and comment or delete line 41

```
if (session_status() !== PHP_SESSION_ACTIVE) session_start();
```

to 

```
//if (session_status() !== PHP_SESSION_ACTIVE) session_start();
```
Probably the php configuration on your hosting account has session autostart or a auto_prepend_file that starts the session.

I have a shared hosting account where I test and usually find bugs that are not present on dev but there are many different configurations on shared hosting accounts that can cause different issues.

**Comment by kvirtu** [comment]
By default a link to /admin/index.html, which is empty. If /admin/index.php - a prompt appears to enter a username and password.
But after entering the login and password - a white screen, with an error

**Reply by givanz** [reply]
There should be an error message in the logs, the copy pasted text looks like it's from apache access.log instead of php error log, probably the php error log is at another location.

You can edit /index.php and add `phpinfo();` at the beginning (or replace the contents of the entire file) to show the path to error_log 

```php
<?php
namespace Vvveb;
phpinfo();
```

**Reply by givanz** [reply]
In the zip there is no /public/admin/index.html there is one at /admin/index.html that I forgot to delete, please delete the file and see if it works.

**Comment by kvirtu** [comment]
error:
89.209.76.211 - - [22/Dec/2022:19:24:54 +0200] "GET /admin/index.php HTTP/1.1" 200 3794 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36"
[22-Dec-2022 19:24:55 Europe/Kiev] SELECT  *,
                 pd.name AS name,
                 products.image

        
     ,(SELECT CONCAT('[', GROUP_CONCAT('{"id":"', pi.product_image_id, '","image":"' , pi.image, '"}'), ']') FROM product_image as pi WHERE pi.product_id = products.product_id GROUP BY pi.product_id) as images
    

    
   FROM product AS products
   
    LEFT JOIN product_to_site p2s ON (products.product_id = p2s.product_id)
    LEFT JOIN product_description pd ON (products.product_id = pd.product_id)  

        
    
    
    WHERE pd.language_id = 1 AND p2s.site_id = 1
    
               
     AND products.type = "product"
         
    
   LIMIT 0, 8
[22-Dec-2022 19:24:55 Europe/Kiev] SELECT count(*) FROM (
    
      SELECT products.product_id FROM product AS products
   
    LEFT JOIN product_to_site p2s ON (products.product_id = p2s.product_id)
    LEFT JOIN product_description pd ON (products.product_id = pd.product_id)  

        
    
    
    WHERE pd.language_id = 1 AND p2s.site_id = 1
    
               
     AND products.type = "product"
         
    
    ) as count
89.209.76.211 - - [22/Dec/2022:19:24:55 +0200] "GET /public/admin/favicon.ico HTTP/1.1" 200 62801 "http://mysite/admin/index.php" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36"
89.209.76.211 - - [22/Dec/2022:19:25:15 +0200] "POST /admin//?module=user/login HTTP/1.1" 200 - "http://mysite/admin/index.php" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36"

**Reply by givanz** [reply]
These are executed sql queries that show in the logs because DEBUG is enabled by default, I disabled query debug in the release, you can update /system/db/mysqli.php to get rid of these messages in the log.

For admin I think deleting /admin/index.html should fix it, does the frontend work?

**Comment by kvirtu** [comment]
I removed index.html from admin and now its working ))).
Thanks

**Comment by kvirtu** [comment]
I like your project. Ready to test, there are bugs and wishes. can you give me your e-mail ?

**Comment by givanz** [comment]
Thank you for testing and providing feedback, the project is still in development and there is still work to be done to reach the beta stage.
You can find my email on my profile page.
